run_llama_train.sh

```diff
@@ -7,19 +7,13 @@
 
 set -ex
 
-TRAINER_DIR=${TRAINER_DIR:-/home/$USER/local/torchtitan}
-
 # use envs as local overrides for convenience
 # e.g.
 # LOG_RANK=0,1 NGPU=4 ./run_llama_train.sh
-
 NGPU=${NGPU:-"8"}
 NNODES=${NNODES:-"1"}
-
-# by default log just rank 0 output,
+TRAINER_DIR=${TRAINER_DIR:-/home/$USER/local/torchtitan}
 LOG_RANK=${LOG_RANK:-0}
-
-
 CONFIG_FILE=${CONFIG_FILE:-"./train_configs/debug_model.toml"}
 
 overrides=""
```

Comment:
do we plan to remove the memory estimation script in later PRs?

Reply:
yup :) see the next PR.